src/test/java/net/sf/jabref/logic/exporter/ModsExportFormatTestFiles.java

```diff
@@ -65,7 +62,7 @@ public void setUp() throws Exception {
         charset = StandardCharsets.UTF_8;
         modsExportFormat = new ModsExportFormat();
         tempFile = testFolder.newFile();
-        testImporter = new BibtexImporter(ImportFormatPreferences.fromPreferences(Globals.prefs));
+        testImporter = new BibtexImporter(Globals.prefs.getImportFormatPreferences());
     }
 
     @Test
```

Comment:
Just saw it: Please use Standard.Charsets.UTF-8, otherwise gradle check on windows  will be complaining about the encoding again ;)